_includes/neurips/2020/about.neurips.html

```diff
@@ -1,7 +1,7 @@
 <div class="about-details" style="font-size: 18px">
   <h2> LatinX in AI Research at NeurIPS 2020 </h2>
   <section class="workshop-intro" id="workshop-intro" style="color: black">
-    <p>The LXAI Workshop at <a target="_blank"
```

Comment:
Please do not remove the target="_blank" for external links - this allows the link to open in a new tab.